src/os_win/os_fs.c

```diff
@@ -361,8 +361,8 @@ __win_handle_read(
 		if (!ReadFile(fh->filehandle, addr, chunk, &nr, &overlapped))
 			WT_RET_MSG(session,
 			    nr == 0 ? WT_ERROR : __wt_getlasterror(),
-			    "%s: handle-read: ReadFile: failed to read %"
-			    WT_SIZET_FMT " bytes at offset %" PRIuMAX,
+			    "%s: handle-read: ReadFile: failed to read %lu "
```

Comment:
D'oh. Thanks @keithbostic I misread the description in the ticket - I'd thought a capital "i" was an "l".